revenue/revenueapp/templates/revenueapp/review_form.html

```diff
@@ -14,34 +14,35 @@
  <img src= "{% static "media/ReVENUE_logo.png" %}" alt="home" width="25%" class = "center"/>  
   </div>
 
-<body id="background">
-    
-<div class="navbar">
-   <ul>
-      <li><a href="{% url 'home' %}">Home</a></li>
-      <li><a href="{% url 'about' %}">About</a></li>
-      <li><a href="{% url 'venue_create' %}">Create A Venue</a></li>
-      <li><a href="{% url 'logout' %}?next={% url 'individual_venue' venue.id %}">Logout</a></li>
-
-   </ul>
-</div>
-
-<br>
-<br>
-
-<img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcR6sdT7cf98F4_QUJiEqs14XwqNaE-vZPzqRQ&usqp=CAU" alt="10 best venues in houston.com" width="60%" class="center">
-
-<h1 id="create_review_heading_text">Please leave a review</h1>
-
-<h1> {{ venue.name }} </h1>
-<form method="post">
-    {% csrf_token %}
-    {{ form.as_p }}
-    <input type="submit" value="Save">
-    <a href="{% url 'individual_venue' venue.id %}">Cancel</a>
```

Comment:
I also added a cancel button and the review's  name to the form, and it has been overwritten with an older version of this form.